db/migrate/20151217211019_create_spotlight_exhibit_filters.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSpotlightExhibitFilters < ActiveRecord::Migration
+  def change
+    create_table :spotlight_exhibit_filters do |t|
+      t.string :field
+      t.string :value
+      t.references :spotlight_exhibit, index: true, foreign_key: true
```

Comment:
I think if this was just `references :exhibit`, everything would work out automagically?